udata/core/dataset/models.py

```diff
@@ -83,7 +83,15 @@ def __unicode__(self):
         return self.title
 
 
-class DatasetQuerySet(db.BaseQuerySet):
+class OwnedByQuerySet(db.BaseQuerySet):
```

Comment:
As the same pattern is used by `Reuse`, maybe it should be extracted as a mixin into `udata/models/__init__.py` so we could write something like:
```python
class OwnedByQuerySet(db.BaseQuerySet, db.OwnedQuerySetMixin):
    pass

class DatasetQuerySet(db.BaseQuerySet, db.OwnedQuerySetMixin):
    def visible(self):
        # ...
```